docs/source/guide/tutorials/base/how_to_train/instance_segmentation.rst

```diff
@@ -1,2 +1,327 @@
+an OTX-workspace first.
+
+Let's prepare an OTX instance segmentation workspase running the following command:
+
+.. code-block::
+
+  (otx) ...$ otx build --task instance_segmentation --model MaskRCNN-ResNet50
+
+  [*] Workspace Path: otx-workspace-INSTANCE_SEGMENTATION
+  [*] Load Model Template ID: Custom_Counting_Instance_Segmentation_MaskRCNN_ResNet50
+  [*] Load Model Name: MaskRCNN-ResNet50
+  [*]     - Updated: otx-workspace-INSTANCE_SEGMENTATION/model.py
+  [*]     - Updated: otx-workspace-INSTANCE_SEGMENTATION/data_pipeline.py
+  [*]     - Updated: otx-workspace-INSTANCE_SEGMENTATION/tile_pipeline.py
+  [*]     - Updated: otx-workspace-INSTANCE_SEGMENTATION/deployment.py
+  [*]     - Updated: otx-workspace-INSTANCE_SEGMENTATION/hpo_config.yaml
+  [*]     - Updated: otx-workspace-INSTANCE_SEGMENTATION/compression_config.json
+  [*] Update data configuration file to: otx-workspace-INSTANCE_SEGMENTATION/data.yaml
+
+  (otx) ...$ cd ./otx-workspace-INSTANCE_SEGMENTATION
+
+It will create **otx-workspace-INSTANCE_SEGMENTATION** with all necessery configs for MobileNet-V3-large-1x, prepared ``data.yaml`` to simplify CLI commands launch and splitted dataset.
+
+.. note::
+  Using ``otx train`` with TEMPLATE allows you to run the training directly without ``otx build``.
+
+  However, this requires ``--train-data-roots`` and ``--val-data-roots`` in the command.
+
+  .. code-block::
+
+    (otx) ...$ otx train MaskRCNN-ResNet50 \
+                      --train-data-roots  /home/<username>/training_extensions/tests/assets/car_tree_bug \
+                      --val-data-roots /home/<username>/training_extensions/tests/assets/car_tree_bug \
+                      params --learning_parameters.num_iters 8
+
+  The above command also creates an ``otx-workspace-INSTANCE_SEGMENTATION``, just like running build. This also updates ``data.yaml`` with data-specific commands.
+
+  For more information, see :doc:`quick start guide <../../../get_started/quick_start_guide/cli_commands>`
+
+3. Next, we need to create 
+train/validation sets. 
+
+To simplify the command line functions calling, we may create a ``data.yaml`` file with annotations info and pass it as a ``--data`` parameter. The content of the ``otx-workspace-INSTANCE_SEGMENTATION/data.yaml`` for dataset should have absolute paths and will be similar to that:
+
+.. note::
+  You can modify the required arguments in ``data.yaml`` or use the command to provide the required arguments.
+
+.. code-block::
+
+  {'data':
+    {
+    'train':
+      {'data-roots': '/home/<username>/training_extensions/tests/assets/car_tree_bug'},
+    'val':
+      {'data-roots': '/home/<username>/training_extensions/tests/assets/car_tree_bug'},
+    'test':
+      {'data-roots': '/home/<username>/training_extensions/tests/assets/car_tree_bug'}
+    }
+  }
+
+``Ann-files`` contains a path to the annotation, while ``data-roots`` is a path to the folder, where images are stored.
+
+4. To start training we need 
```

Comment:
I think there some mismatch in consequence commands. Let's use only one ``otx train`` command example.
Also, I would suggest refer to https://github.com/openvinotoolkit/training_extensions/blob/develop/docs/source/guide/tutorials/base/how_to_train/classification.rst more then OD tutorial, since they are different. We intended to make OD more deep while other tutorials should be like live examples on specific dataset

Reply:
I found that the OD tutorial didn't work for me, so I mainly referred to the classification tutorial.

Reply:
I was just mislead by many "otx train" commands with and without otx workspace